src/view/provider/history/HistoryProvider.tsx

```diff
@@ -6,7 +6,7 @@ interface Props {
 }
 
 export type HistoryContextType = {
-    calculationHistory: CalculationData[];
+    calculationHistory: CalculationData[] | null;
```

Comment:
why not an empty array? 👀

Reply:
I was thinking of making it nullable so that I don't access array methods when `calculationHistory` is null, 
e.g. here we are checking calculationHistory is not null before calling calculationHistory[0] 
```typescript
// Results.tsx
return (
        calculationHistory && (
            <div className=" h-32 grid grid-cols-2 text-base bg-nyanza rounded-2xl shadow font-medium">
                <EmissionsSummary />
                <EmissionsComparison calculation={calculationHistory[0]} />
            </div>
        )
    );
```
e.g. also here if it is an empty array, `calculation.emissions` will throw and error
```typescript
export const EmissionsComparison = ({
    calculation,
}: {
    calculation: CalculationData;
}) => {
    ...
    const comparison = getEmissionsComparison(calculation.emissions);
    return (
```